app/models/target_group.rb

```diff
@@ -0,0 +1,5 @@
+class TargetGroup < ApplicationRecord
+	has_many :preferred_target_groups
+
+	validates :name, presence: true 
```

Comment:
we should also validate that it is unique -- `uniqueness: { case_sensitive: false }`